services/housekeeper/housekeeper.go

```diff
@@ -248,6 +273,48 @@ func (hk *Housekeeper) UpdateProposerDutiesWithoutChecks(headSlot uint64) {
 	log.WithField("numDuties", len(_duties)).Infof("proposer duties updated: %s", strings.Join(_duties, ", "))
 }
 
+// When the update is triggered, we store registered validators in Redis for an the current and next epoch.
```

Comment:
Per conversation today this should just be for the current epoch, correct?

Reply:
The relay already employees this as their strategy, I'm guessing it's so that at the start of the epoch they still have some semblance of which validator needs to go next. Though this is def a confusing thing for me. Because the relay auction needs to run before the slot starts. That means a validator with a proposer duty to produce at slot 0 of the next epoch, must run the auction in the previous epoch. But how would that be possible, given that we've shown the proposer duties are only stable in the current epoch.